codalab/server/bundle_manager.py

```diff
@@ -263,7 +265,10 @@ def _make_bundle(self, bundle):
 
                     # If source path is on Azure Blob Storage, we should download it to a temporary local directory first.
                     if parse_linked_bundle_url(dependency_path).uses_beam:
-                        dependency_path = os.path.join(tempdir, dep.parent_uuid)
+                        if dep.child_path != "":
```

Comment:
What does this code do?

Reply:
The  `dep.child_path` is the key specified by user, eg `cl make key1:[bundle_id1] --store azure`.  I want to use the key as the new sub-folder name if the user are making multiple bundle into one folder.

Reply:
Let's replace this whole if statement with

```
dependency_path = os.path.join(tempdir, dep.child_path or dep.parent_uuid)
```

And add a comment explaining when we would use dep.child_path

Reply:
@wwwjn can you update this?